src/lib/recipes.ts

```diff
@@ -422,6 +422,104 @@ const xiphosRecipes: IRecipe[] = [
   ),
 ];
 
+// HH-a "Javelin" half-truck recipes
+const javelinRecipes: IRecipe[] = [
+  // Garage production: 20 x Refined Materials → 1 x HH-a "Javelin"
+  createRecipe(
+    [{ stuff: Materials.RefinedMaterials, count: 20 }],
+    [{ stuff: Vehicles.Javelin, count: 1 }]
+  ),
+  // Mass Production Factory: 150 x Refined Materials → 3 crates of 3 x HH-a "Javelin" (9 total)
+  createRecipe(
+    [{ stuff: Materials.RefinedMaterials, count: 150 }],
+    [{ stuff: Vehicles.Javelin, count: 9 }]
+  ),
+];
+
+// HH-b "Hoplite" half-truck recipes
+const hopliteRecipes: IRecipe[] = [
```

Comment:
Only one recipe. Requirements for one hoplite:
3 x [Processed Construction Materials](https://foxhole.wiki.gg/wiki/Processed_Construction_Materials) ProcessedConstructionMaterialsIcon.png
3 x [Assembly Materials IV](https://foxhole.wiki.gg/wiki/Assembly_Materials_IV) AssemblyMaterials4Icon.png
[HH-a “Javelin”](https://foxhole.wiki.gg/wiki/HH-a_%E2%80%9CJavelin%E2%80%9D) HalfTrackColVehicleIcon.png